nvtabular/tags.py

```diff
@@ -13,11 +13,24 @@ class Tags(Enum):
     # Feature context
     USER = "user"
     ITEM = "item"
-    ITEM_ID = ["item", "item_id"]
+    ID_ITEM = "item_id"
     CONTEXT = "context"
 
+    BINARY = "binary"
+    REGRESSION = "regression"
+    MULTI_CLASS = "multi_class"
+
     # Target related
-    TARGETS = ["target"]
-    TARGETS_BINARY = ["target", "binary"]
-    TARGETS_REGRESSION = ["target", "regression"]
-    TARGETS_MULTI_CLASS = ["target", "multi_class"]
+    TARGETS = "target"
+
+    @property
+    def TARGETS_BINARY(self):
```

Comment:
I was thinking of these as lower-cased method names. Now that I see it this way, I'm not sure which way makes more sense. Do we want to give the impression they're interchangeable with the singular tags or not? 🤔

Reply:
I don't think using a property here is going to work - properties work on a instance of a class, and usage here will be on the class itself.

Like running this

```
print(repr(Tags.TARGETS_BINARY))
print(repr(Tags.TIME))
```

prints out 
```
<property object at 0x7fdc684c3cc0>
<Tags.TIME: 'time'>
```

Reply:
Good point! They'd have to be class methods.